scripts/k8s-storage/driver-rbd-rwo.yaml

```diff
@@ -0,0 +1,72 @@
+---
+ShortName: cephcsi-rbd-rwo-test
```

Comment:
why its RWO? can we have a generic name as we keep on  adding more testing later

Reply:
I want to add a rwx driver too, one that disables Filesystem-mode for PVCs (not sure yet how to do that)